src/site/content/en/blog/introducing-model-viewer/index.md

```diff
@@ -0,0 +1,139 @@
+experiences to customers' homes. Museums are making 3D models of their artifacts
+available to everyone on the web. Unfortunately, it can be difficult to add a 3D
+model to a website in a way that provides a great user experience without a deep
+knowledge of 3D technologies or resorting to hosting 3D content on a third-party
+site. The `<model-viewer>` web component, [introduced in early
+2019](../model-viewer), seeks to
+make putting 3D models on the web as easy as writing a few lines of HTML. Since
+then, the team has been working to address feedback and requests from the
+community. The culmination of that work is `<model-viewer>` version 1.0.
+
+## What's new since last year?
+
+Version 1.0 includes built-in support for augmented reality (AR) on the web,
+improvements to speed and fidelity, and other frequently-requested features.
+
+### Augmented reality
+
+Viewing a 3D model on a blank canvas is great, but being able to view it in your
+space is even better. Now `<model-viewer>` supports [WebXR augmented
+reality](https://modelviewer.dev/examples/webxr.html0), for an
+entirely-within-the-browser 3D and AR experience on Chrome Android.
+
+```html
+<model-viewer src="ToyTrain.glb" ar ar-scale="auto" camera-controls alt="A 3D model of a wooden toy train">
+</model-viewer>
+```
+<iframe style="width:100%; height: 100%;position: absolute; top: 50%; left: 50%; transform: translate(-50%,-50%);" src="https://modelviewer.dev/webdotdev/webxr.html" frameborder="0" allowfullscreen></iframe>
+
+The above example shows the minimum required to use AR. The `<model-viewer>`
+component also allows you use HTML to customize the WebXR AR experience, as
+shown in [this example on the modelviewer.dev
+site](https://modelviewer.dev/examples/webxr.html).
+
+### Camera controls
+
+`<model-viewer>` now gives full control over the view's virtual camera (the
+perspective of the viewer). This includes the camera target, orbit (position
+relative to the model), and field of view. You can also enable auto-rotation and
+set limits on user interaction (e.g. maximum and minimum fields of view).
+
+### Annotations
+
+You can also annotate your models using HTML and CSS. This capability is often
+used to "attach" labels to parts of the model in a way that moves with the model
+as it's manipulated. The annotations are customizable, including their
+appearance and the extent to which they're hidden when they're positioned behind
+the model. Annotations also work in AR.
+
+```html
+<style>
+  button{
+    display: block;
+    width: 6px;
+    height: 6px;
+    border-radius: 3px;
+    border: 3px solid blue;
+    background-color: blue;
+    box-sizing: border-box;
+  }
+
+  #annotation{
+    background-color: #dddddd;
+    position: absolute;
+    transform: translate(10px, 10px);
+    border-radius: 10px;
+    padding: 10px;
+  }
+
+<model-viewer src="https://modelviewer.dev/assets/ShopifyModels/ToyTrain.glb" alt="A 3D model of a Toy Train" camera-controls>
```

Comment:
Should put a Shopify copyright in if we're going to use their models.

Reply:
It's a code example. The article doesn't actually use the model. Though the <model-viewer> website does. https://modelviewer.dev/examples/webxr.html